packages/krino/krino/krino_lib/Akri_BoundingBoxMesh.hpp

```diff
@@ -75,7 +76,7 @@ class BoundingBoxMesh {
 public:
   BoundingBoxMesh(stk::topology element_topology, const std::vector<std::string>& rank_names = std::vector<std::string>());
   void set_domain(const BoundingBoxType & mesh_bbox, const double mesh_size, const int pad_cells = 0);
-  void populate_mesh(stk::ParallelMachine pm = MPI_COMM_WORLD, const stk::mesh::BulkData::AutomaticAuraOption auto_aura_option = stk::mesh::BulkData::AUTO_AURA);
```

Comment:
I think having MPI_COMM_WORLD as default value is ok, if the user wants, he'll just use different comm